packages/scandipwa/src/route/CartPage/CartPage.component.js

```diff
@@ -147,13 +147,15 @@ export class CartPage extends PureComponent {
         const {
             totals: {
                 tax_amount = 0,
+                shipping_tax_amount = 0,
                 cart_display_config: {
                     display_zero_tax_subtotal
                 } = {}
             }
         } = this.props;
+        const tax = tax_amount - shipping_tax_amount;
```

Comment:
Is it possible to retrieve this from BE properly? The less calculations we do on FE and the more correct data we fetch from BE - the better.